app/assets/stylesheets/runtime-banner.scss

```diff
@@ -12,7 +12,8 @@
   flex-flow: row nowrap;
   align-items: center;
   padding: var(--su-3);
-  max-width: calc(min(100% - var(--su-4) * 2, 750px));
+  max-width: 750px;
+  width: calc(100% - var(--su-4) * 2);
```

Comment:
@fdoxyz does this look like it captures the intended sizing from https://github.com/forem/forem/pull/13662

Reply:
I tried it out locally and didn't quite work (overflowed to the right)

![IMG_F5A9BA1765D6-1](https://user-images.githubusercontent.com/6045239/123342002-2a3e9380-d50c-11eb-9f25-47fbc8a767a2.jpeg)

I think that the flex styling makes the banner grow to fill the container by default, that's why it only had `max-width` before. We're refactoring the way it works but the design [will remain the same for now](https://github.com/forem/rfcs/pull/249#discussion_r658085600) 🙂

Reply:
I did see this error but CI didn't fail (the job itself). I wonder if the `min` command is the problem here though 🤔

Reply:
Yes, the `calc(min(...))` combination seems to be the parse problem (sass passes it through and postcss rejects it) - the real problem with this setup seems to be that postcss failing to process doesn't cause the build to fail (basically whatever processing we assumed was happening can fail silently and the build proceeds) - you get no feedback if you break something, and we expect that.

It's also possible that the ios browser's css rendering accepts this just fine and _only_ postcss's parser is problematic, that was one of the reasons I started by removing cruft from the configuration and then moved on to finding the problematic line. 

Thanks so much for spot checking, I'll see if there's some other way to handle this correctly.